OmarEspinosaCastro/26-Agosto/HerenciaPolimorfismoEncapsulamiento/Principal.java

```diff
@@ -0,0 +1,28 @@
+
+public class Principal {
+
+    public static void main(String[] args) {
+
+        Job[] jobs = new Job[3];
+        jobs[0] = new Driver("Juan", 30, 1000.0, 5);
+        jobs[1] = new Doctor("Pedro", 55, 20000, "Cirujano");
+        jobs[2] = new Teacher("Pablo", 30, 150, 20);
+
+        for (Job job : jobs) {
+            salary(job);
+        }
+
+    }
+
+    public static void salary(Job job) {
+        if (job instanceof Driver) {
+            ((Driver) job).calculateSalary();
+            System.out.println(job.getName() + " para el puesto de Chofer tu saldo es: " + job.getSalary());
```

Comment:
Maybe to avoid write in a hard code the position of job like "chofer, médico, docente" you can add another property in a Job object like String position and make toString method in order you can get it in you System.out.println

Reply:
the position was added as an attribute and I create the method to print